File: MercadoPagoSDKExamplesObjectiveC/MercadoPagoSDKExamplesObjectiveC/CustomTableViewCell.m
//
//  CustomTableViewCell.m
//  MercadoPagoSDKExamplesObjectiveC
//
//  Created by Maria cristina rodriguez on 2/8/17.
//  Copyright © 2017 MercadoPago. All rights reserved.
//

#import "CustomTableViewCell.h"
@import MercadoPagoSDK;

@implementation CustomTableViewCell 

-(UINib *)getNib {
    return [UINib nibWithNibName:@"CustomTableViewCell" bundle: [NSBundle mainBundle]];
}

-(CGFloat)getHeigth {
    return (CGFloat)180;
}

@end

File: MercadoPagoSDKExamples/MercadoPagoSDKExamples/CustomTableViewCell.swift
//
//  CustomTableViewCell.swift
//  MercadoPagoSDKExamples
//
//  Created by Eden Torres on 2/6/17.
//  Copyright © 2017 MercadoPago. All rights reserved.
//

import UIKit
import MercadoPagoSDK

open class CustomTableViewCell: MPCustomTableViewCell {
    //var imageDelegate: bundle.CellProtocol?
    @IBOutlet open weak var title: UILabel!

    override open func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }
    
    func fillCell(text: String){
        title.text = text
    }

    override open func setSelected(_ selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }
    
}
